custom/icds_reports/migrations/0156_auto_20191217_0932.py

```diff
@@ -54,7 +54,7 @@ class Migration(migrations.Migration):
         ),
         migrations.AlterUniqueTogether(
             name='aggregateadolescentgirlsregistrationforms',
-            unique_together=set([('supervisor_id', 'person_case_id', 'month')]),
```

Comment:
since this migration was already run on at least one environment, you will need to make a new migration to alter that index as this will not be run on india again. furthermore, you should confirm that the migration does what you expect as im not sure if django will automatically handle reordering in the way you intend.

Reply:
Yeah sure.